src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,217 @@
+            Donations
+          </Link>
+          <Link to="/about" className="eachLinksInNavbar">
+            About
+          </Link>
+          <Link to="/contactUs" className="eachLinksInNavbar">
+            Contact Us
+          </Link>
+          <DropdownButton id="dropdown-basic-button" title="Language">
+            <Dropdown.Item href="#/action-1">TUR</Dropdown.Item>
+            <Dropdown.Item href="#/action-2">ENG</Dropdown.Item>
+          </DropdownButton>
+        </div>
+        <div id="navbarProfile">
+          <Link to="/logIn">
+            <Button
+              variant="primary"
+              style={{ borderRadius: "0.75rem", marginBottom: "5%" }}
+            >
+              Log In
+            </Button>
+          </Link>
+          <Link to="/signUp">
+            <Button
+              variant="primary"
+              style={{ borderRadius: "0.75rem", marginBottom: "5%" }}
+            >
+              Sign Up
+            </Button>
+          </Link>
+        </div>
+      </div>
+    </Router>
+  );
+};
+
+const NavbarForSmallScreen = () => {
+  const [sidebar, setSidebar] = useState(false);
+
+  const showSidebar = () => setSidebar(!sidebar);
+
+  const boxRef = useRef(null);
+
+  useEffect(() => {
+    const handleClickOutside = (event) => {
+      if (boxRef && boxRef.current.contains(event.target)) {
+        setSidebar(true);
+      } else {
+        setSidebar(false);
+      }
+    };
+
+    document.addEventListener("mousedown", handleClickOutside);
+    return () => {
+      document.removeEventListener("mousedown", handleClickOutside);
+    };
+  }, boxRef);
+
+  return (
+    <>
+      <div className="navbar-mobile">
+        <Link to="#" className="menu-bars">
+          <List onClick={showSidebar} />
+        </Link>
+        <Link to="/" className="navbar-mobile-logo">
+          LOGO
+        </Link>
+      </div>
+      <nav ref={boxRef} className={sidebar ? "nav-menu active" : "nav-menu"}>
+        <ul className="nav-menu-items">
+          <li className="navbar-toggle" onClick={showSidebar}>
+            <Link to="#" className="menu-bars">
+              <List />
+            </Link>
+          </li>
+
+          <li className="nav-li nav-li-login-sign" onClick={showSidebar}>
+            <Link to="/logIn">
+              <Button
+                variant="primary"
+                style={{ borderRadius: "0.75rem", marginBottom: "5%" }}
+              >
+                Log In
+              </Button>
+            </Link>
+          </li>
+          <li className="nav-li nav-li-login-sign" onClick={showSidebar}>
+            <Link to="/signUp">
+              <Button
+                variant="primary"
+                style={{ borderRadius: "0.75rem", marginBottom: "5%" }}
+              >
+                Sign Up
+              </Button>
+            </Link>
+          </li>
+
+          <li className="nav-li" onClick={showSidebar}>
```

Comment:
repetetion, you can use an array of objects to iterate over it and create each item alone

Reply:
Hey @halitbatur I have problem and couldnt solve it. Can you help me?
Problem: bootstrap-icons are component and I can not apply map function for components. 


![resim](https://user-images.githubusercontent.com/44509662/129283751-60b262a9-d389-4d8f-a1a7-fa4521afe505.png)

![resim](https://user-images.githubusercontent.com/44509662/129283776-8bbc6751-b179-4db1-adac-a8be323edac0.png)